math/eigen/include/algebra/math/impl/eigen_transform3.hpp

```diff
@@ -153,8 +153,14 @@ struct transform3 {
     _data_inv = _data.inverse();
   }
 
+  /** Default constructor: set contents to identity matrices */
+  ALGEBRA_HOST_DEVICE
+  transform3() {
+    _data.setIdentity();
+    _data_inv.setIdentity();
+  };
```

Comment:
Something is fishy with the warning configuration in this project. :confused: This semicolon really should've triggered one. :thinking: (I was removing such extra semicolons left and right from detray today.)

Reply:
Since I'm keen to make a new tag of the project for detray's benefit, I decided to be a bit aggressive, and removed the semicolon myself.

Reply:
Sure, works for me. :smile: